src/lib/stores/settings-store.ts

```
@@ -0,0 +1,74 @@
+		};
+	});
+}
+
+// Function to get current settings value
+export function getSettings(): Settings {
+	let currentSettings: Settings;
+	const unsubscribe = settingsStore.subscribe((value) => {
+		currentSettings = value;
+	});
+	unsubscribe();
+	return currentSettings!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)